### PR TITLE
Add xk6-nostr

### PR DESCRIPTION
**Pull request by akiomik**
Hi, I created an extension to interact with Nostr relays.
This extension is useful for relay load testing.

- https://github.com/akiomik/xk6-nostr

NOTE: Nostr is a protocol that mainly designed for distributed social media. Nostr uses websockets for communication between clients and servers (relays).

ref:

- http://nostr.com
- https://en.wikipedia.org/wiki/Nostr

### CODE REVIEW

**Comment by heitortsergent** [comment]
@akiomik thanks for creating this extension! I'll get someone on the team to take a look at it. 🙇

File: src/data/doc-extensions/extensions.json [hunk 1]
```diff
@@ -1141,6 +1141,21 @@
...
       "cloudEnabled": false
+    },
+    {
+      "name": "xk6-nostr",
+      "description": "Interact with Nostr relays",
+      "url": "https://github.com/akiomik/xk6-nostr",
+      "logo": "",
+      "author": {
+        "name": "Akiomi Kamakura",
+        "url": "https://github.com/nostr"
```
Comment (javaducky) [inline src/data/doc-extensions/extensions.json:1152]:
```suggestion
        "url": "https://github.com/akiomik"
```

**Review by javaducky — CHANGES_REQUESTED** [review]
Just one minor change for the author url.

File: src/data/doc-extensions/extensions.json (hunk 1 above)
Comment (akiomik) [reply src/data/doc-extensions/extensions.json:1152]: Thanks👍

**Review by javaducky — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**